### ISSUE TITLE
Search in more than one attributes 

### ISSUE THREAD

**Issue by ahmed616** [opensourcepos/opensourcepos#2407]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 
- OSPOS git commit hash is: 
- PHP version is: 
- MySQL or MariaDB version is: 
- OS and version is: 
- WebServer is: 
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### Issue 
 When search in more than one attribute there is no result shown and when search in one attribute the value of other attributes disappear.

![1](https://user-images.githubusercontent.com/44599622/54476080-cd6ab380-4801-11e9-8db7-9740627876d2.PNG)
![3](https://user-images.githubusercontent.com/44599622/54476081-cd6ab380-4801-11e9-8e37-bd6b1f795e75.PNG)
![2](https://user-images.githubusercontent.com/44599622/54476082-ce034a00-4801-11e9-9cc8-3610abe40141.PNG)

**Comment by jekkos** [comment]
Something for me to look at.

**Comment by daN4cat** [comment]
@jekkos shall we move this to 3.4.0?

**Comment by jekkos** [comment]
Not sure but I think this one can be closed as well now.

**Comment by odiea** [comment]
Tables go away if I search on more than 1 attribute.

**Comment by jekkos** [comment]
How do you mean exactly? Is it the data in other columns that disappears?

**Comment by odiea** [comment]
The 1st image has a search for 1 attribute. The second image I added an additional attribute. I assume this is what hew was trying to do?
![2019-06-30 14_51_47-Window](https://user-images.githubusercontent.com/10962177/60402017-c2d0f180-9b46-11e9-9552-0789f24e5adb.png)
![2019-06-30 14_52_14-Window](https://user-images.githubusercontent.com/10962177/60402018-c95f6900-9b46-11e9-825e-b97812690f92.png)

**Comment by jekkos** [comment]
Ok now I see, I wonder whether that even works with regular search?

**Comment by jekkos** [comment]
So saying that this might be a non issue. I might change this into a feature request as I think that doesn't even work with regular item search. Any opinions?

**Comment by daN4cat** [comment]
I would say that aggregate search on multiple attributes is a new feature fro 3.4.0.

**Comment by daN4cat** [comment]
We should probably support a search syntax like: 

`tag color: blue, condition: new`

**Comment by jekkos** [comment]
Yes some kind of search DSL would be nice here, but indeed lets move to 3.4 as I don't have a direct urge to implement it.
